src/multiParamManager.js

```diff
@@ -96,11 +114,17 @@ class MultiParamManager {
     }
   }
 
-  createMultiFields () {
+  createMultiFields (uniqueId) {
     if (this.funABI.inputs) {
+      var that = this
       return yo`<div>
-        ${this.funABI.inputs.map(function (inp) {
-          return yo`<div class="${css.multiArg}"><label for="${inp.name}"> ${inp.name}: </label><input placeholder="${inp.type}" title="${inp.name}"></div>`
+        ${this.funABI.inputs.map(function (inp, idx) {
+          function setUpValidator (id) {
+            $(`#${id}`).off('blur')
+            $(`#${id}`).blur(() => doValidation(that, id))
+          }
+          var id = `${uniqueId}${inp.type}${inp.name}${idx}`
+          return yo`<div class="${css.multiArg}"><label for="${inp.name}"> ${inp.name}: </label><input id="${id}" onclick=${() => { setUpValidator(id) }} placeholder="${inp.type}" title="${inp.name}"></div>`
```

Comment:
Perhaps using `onchange` event instead of onclick would be better for this? 
And to avoid spamming perhaps having a mechanism like https://github.com/ethereum/remix-ide/blob/master/src/app.js#L941
which start validating after like 700ms of inactivity

Reply:
That depends - is the desired behavior that we are validating as someone is typing or after they have left the text box? If it's while they are typing, then I can alter this to work according to `oninput`; however, the `onchange` event isn't actually fired until after you have left the textbox and so will offer the same functionality as what is already here (where the event listener is set up on the initial `onclick`).

Reply:
ah ok I see, the thing with "onclick" is that the validation start before the user start typing.
so `oninput` is definitely better than `onchange`

Reply:
oh ok sry i didn't see that.
so yes let's keep the `onclick`.
could the red border be in a different color / thickness then when the input has the focus. In chrome that's almost the same

Reply:
yes for sure I can make it thicker, right now it gets covered by the highlight